zcash_client_backend/Cargo.toml

```diff
@@ -13,11 +13,13 @@ license = "MIT OR Apache-2.0"
 edition = "2018"
 
 [dependencies]
+async-trait = "0.1.52"
 bech32 = "0.9.1"
 bls12_381 = "0.3.1"
 bs58 = { version = "0.4", features = ["check"] }
 base64 = "0.13"
 ff = "0.8"
+futures = { version = "0.3", package = "futures", features = ["compat", "async-await"] }
```

Comment:
Q: Why package field was used here?

Reply:
was moving this crate definitions from mm2 so I forgot to adjust appropriately. fixed

Reply:
@borngraced It's still there :slightly_smiling_face:

Reply:
removed